packages/dd-trace/src/opentracing/span.js

```diff
@@ -3,7 +3,7 @@
 // TODO (new internal tracer): use DC events for lifecycle metrics and test them
 
 const opentracing = require('opentracing')
-const now = require('performance-now')
+const { now, dateNow } = require('./time')
```

Comment:
Couldn't the new file be replaced by simply doing this instead in this file:

```js
const now = require('performance-now')
const dateNow = Date.now
```

If more is needed maybe let's just remove `performance-now` entirely as it was mostly added for browser support which is no longer needed.

Reply:
Node has `perf_hooks.performance.now()`, not sure if that's actually equivalent but worth looking into. Otherwise we're guaranteed to always have `hrtime` in Node so the fallbacks are not needed, so it become a 2 liner to just add the seconds to the nanoseconds, no need for a library.

Reply:
yeah that sounds good

Reply:
https://github.com/DataDog/dd-trace-js/pull/2246